src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAvrcpControl.cs

```diff
@@ -0,0 +1,607 @@
+        /// </exception>
+        /// <since_tizen> 8 </since_tizen>
+        public Track GetTrackInfo()
+        {
+            if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
+            {
+                return BluetoothAvrcpControlImpl.Instance.GetTrackInfo();
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+            return null;
+        }
+
+        /// <summary>
+        /// Sends a particular play command to the target device
+        /// </summary>
+        /// <remarks>
+        /// The remote device must be connected.
+        /// </remarks>
+        /// <param name="command">Command to be sent</param>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled
+        /// or when sending command to the target device fails.
+        /// </exception>
+        /// <since_tizen> 8 </since_tizen>
+        public void SendPlayerCommand(PlayerCommand command)
+        {
+            if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
+            {
+                BluetoothAvrcpControlImpl.Instance.SendPlayerCommand(command);
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+        }
+
+        /// <summary>
+        /// Sends a play command to a particular target device
+        /// </summary>
+        /// <remarks>
+        /// The remote device must be connected.
+        /// </remarks>
+        /// <param name="command">Command to be sent</param>
+        /// <param name="remoteAddress">Address of the device to send command</param>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled
+        /// or when sending command to the target device fails.
+        /// </exception>
+        /// <since_tizen> 8 </since_tizen>
+        public void SendPlayerCommandTo(PlayerCommand command, string remoteAddress)
+        {
+            if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
+            {
+                BluetoothAvrcpControlImpl.Instance.SendPlayerCommandTo(command, remoteAddress);
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+        }
+
+        /// <summary>
+        /// Sets absolute volume of target device
+        /// </summary>
+        /// <remarks>
+        /// The remote device must be connected
+        /// </remarks>
+        /// <param name="volume">The volume level to be set(unsigned int)</param>
```

Comment:
What means of `unsigned int` ? 
Volume could be 0 ~ 4,294,967,295?

Reply:
This api is for internal not public. I modified the ACR article.